includes/plugin.php

```diff
@@ -669,6 +671,8 @@ private function __construct() {
 		$this->logger = Log_Manager::instance();
 		$this->data_manager = Data_Manager::instance();
 
+		$this->is_legacy_mode_active = ! empty( get_option( 'elementor_markup_legacy_mode' ) );
```

Comment:
Move the set of this data to its own method, and change the format to an array that can receive multiple values like that:

```
[
    'elementsWrapper' => get_option( 'elementor_markup_legacy_mode' ),
]

Reply:
Why camel case? this is only used in PHP